lang/de_DE.json

```diff
@@ -0,0 +1,416 @@
+    "Page": "Seite",
+    "Page <1></1> of {{totalPages, number}}": "Seite <1></1> von {{totalPages, number}}",
+    "Passwords must match.": "Passwörter müssen übereinstimmen.",
+    "Playable": "Spielbar",
+    "Players": "Spiele",
+    "Please include as much evidence as possible to support your claim. This could include screenshots, videos, links to suspicious profiles, or any other relevant information that demonstrates the alleged violation. Describe each piece of evidence in detail, explaining why it suggests a violation of the rules. The more comprehensive and clear your submission, the more efficiently we can evaluate and address the issue.": "Bitte füge so viele Beweise wie möglich bei, um deine Behauptung zu untermauern. Dazu können Screenshots, Videos, Links zu verdächtigen Profilen oder andere relevante Informationen gehören, die den angeblichen Verstoß belegen. Beschreibe jedes Beweisstück detailliert und erkläre, warum es auf einen Verstoß gegen die Regeln hindeutet. Je umfassender und klarer deine Angaben sind, desto effizienter können wir die Angelegenheit bewerten und angehen.",
+    "Points": "Punkte",
+    "Points Earned Since {{date}}": "Erreichte Punkte seit {{date}}",
+    "Preferences": "Präferenzen",
+    "Previous": "Vorherig",
+    "Previous {{count, number}}": "Vorherige {{count, number}}",
+    "Privacy Policy": "Datenschutzerklärung",
+    "Profile": "Profil",
+    "Progress": "Fortschritt",
+    "Progress was reset successfully.": "Fortschritt wurde erfolgreich zurückgesetzt.",
+    "Proposing a new article idea.": "Eine neue Idee für einen Artikel vorschlagen.",
+    "Proposing art updates.": "Zeichen-Updates vorschlagen.",
+    "Quality Assurance": "Qualitätssicherung",
+    "Questions about art-related rule changes.": "Fragen zu kunstbezogenen Regeländerungen.",
+    "RAEvents": "RAEvents",
+    "RANews": "RANews",
+    "RSS Feeds": "RSS Feeds",
+    "Rarity": "Seltenheit",
+    "Recent Forum Posts": "Neusete Forenposts",
+    "Recent Posts": "Neueste Posts",
+    "Registered Players": "Registrierte Spieler",
+    "Release Date": "Erscheinungsdatum",
+    "Remove from Want to Play Games": "Von deiner persöhnlichen Liste löschen",
+    "Removed {{gameTitle}} from playlist!": "{{gameTitle}} wurde von der Liste entfernt!",
+    "Removed {{gameTitle}}!": "{{gameTitle}} wurde gelöscht!",
+    "Removing...": "Wird gelöscht...",
+    "Reorder Site Awards": "Deine Auszeichnungen der Webseite umordnen",
+    "Report Issue": "Problem melden",
+    "Report Issue - {{achievementTitle}}": "Problem melden - {{achievementTitle}}",
+    "Report to DevCompliance": "Dem DevCompliance melden",
+    "Report to QATeam": "Dem QATeam melden",
+    "Report to WritingTeam": "Dem WritingTeam melden",
+    "Reporting a broken set, leaderboard, or rich presence.": "Melde ein kaputtes Set, Bestenliste, or Status.",
```

Comment:
Should "or" be "oder"?

Reply:
Yeah, will get it fixed

Reply:
Accurate translation would be "Meldung an <Name>"

Reply:
Shouldn't it be somewhat context dependent tho?

Reply:
Decided to go with Spory's suggestion, resolved in latest